azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/impl/BaseLinkHandler.java

```diff
@@ -25,26 +25,38 @@ public BaseLinkHandler(final AmqpLink amqpLink) {
     public void onLinkLocalClose(Event event) {
         final Link link = event.getLink();
         if (TRACE_LOGGER.isInfoEnabled()) {
-            TRACE_LOGGER.info(String.format("linkName[%s]", link.getName()));
+            TRACE_LOGGER.info(String.format("onLinkLocalClose linkName[%s]", link.getName()));
```

Comment:
>onLinkLocalClose [](start = 45, length = 16)

we removed these at some point. having caller method names can be configured in slf4j - for instance logback configfile takes in %m - to spew out method name..

Reply:
We've seen many cases where log files that customers share don't have method name and hence making it part of log message will be helpful and reduce the chance of asking a repro with updated log setting.